packages/database/src/migrations/20200901175734-WishCustomSurveyExportSortByHouseholdId-modifies-data.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const reportId = 'WISH_Custom_Export_Surveys';
+const oldTransformations = [
+  {
+    type: 'ancestorMapping',
+    ancestorType: 'sub_catchment',
+    label: 'Sub Catchment',
+    showInExport: true,
+  },
+  { type: 'transposeMatrix' },
+  { type: 'sortByColumns', columns: [['Sub Catchment'], ['Name'], ['Date']] },
+];
+const newTransformations = [
+  {
+    type: 'ancestorMapping',
+    ancestorType: 'sub_catchment',
+    label: 'Sub Catchment',
+    showInExport: true,
+  },
+  { type: 'transposeMatrix' },
+  {
+    type: 'sortByColumns',
+    columns: [['Sub Catchment'], ['What is the household ID?'], ['Name'], ['Date']],
```

Comment:
Originally intending in this PR to move sortByColumns columns specification to survey config but on second thoughts not sure that is necessary here and maybe better done where the need to have different order configs per survey.
Currently sortByColumns coumns don't apply if said title isn't found in survey so here `'What is the household ID?'` is skipped if question not in survey and next column `name` is next in order.
Happy to revise if it would indeed be worth altering here, else thsi makes this a very simple change.

Reply:
Also specifying the whole transformation config like this was easier due to array, maybe better way to specify?

Reply:
I agree with your approach! I think it's a good, pragmatic call given the circumstances.

Also, it's totally fine (and easy to read)! that you specified the whole array. The only problem would be if another migration would update the same part of the config in parallel, but in this case it's high unlikely and not worth bothering